homework_lesson_5/index_lesson_5_mouse.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title> base page </title>
+    <meta name="author" content="Anna Danielkiewicz">
+    <meta name="description" content="The base page we will be using for learning javascript">
+    <link rel="stylesheet" href="style_lesson_5_mouse.css" type="text/css">    
+  </head>
+  <body>
+
+<header>  
+
+
+  
+  <div class="strangeButton">
+
+   <button id="strangeButton"> Strange button </button>
+
+    <script type="text/javascript">
```

Comment:
script block usually goes directly in html head or body tag. Could you please move it there?